pkg/process/process.go

```diff
@@ -629,6 +629,12 @@ func Load(configBlob []byte) (*config.Config, error) {
 			}
 		}
 
+		if peerData.Gateway != nil {
+			peerData.Gateway = util.Ptr(strings.ReplaceAll(*peerData.Gateway, "-", "_"))
```

Comment:
Can we refactor to assign directly rather than with a `util.Ptr` call? That's really just a helper function for constants.

```go
*peerData.Gateway = strings.ReplaceAll(*peerData.Gateway, "-", "_")
```